rosbag2_samples/api_samples/bag_recorder_nodes/src/data_generator_executable.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+
+#include <rclcpp/rclcpp.hpp>
+#include <rclcpp/serialization.hpp>
+#include <example_interfaces/msg/int32.hpp>
+
+#include <rosbag2_cpp/writer.hpp>
+#include <rosbag2_cpp/writers/sequential_writer.hpp>
+#include <rosbag2_storage/serialized_bag_message.hpp>
+
+#include <iostream>
+#include <memory>
+
+int main(int, char **)
+{
+  example_interfaces::msg::Int32 data;
+  data.data = 0;
+  const rosbag2_cpp::StorageOptions storage_options({"big_synthetic_bag", "sqlite3"});
```

Comment:
As far as I remember using `rosbag2_cpp::StorageOptions` was deprecated and recommended to use   `rosbag2_storage::StorageOptions` instead.

Reply:
Was that change made in Galactic? In Foxy, which these samples target, `storage_options.hpp` is still in `rosbag2_cpp`.

Reply:
It looks like in Galactic. It was completly removed then added back as deprecated in #563